helm/cluster/ci/test-kamaji-control-plane-values.yaml

```diff
@@ -0,0 +1,131 @@
+global:
```

Comment:
this is part of a test i vibecoded out of my ass so the values in here likely dont hold up to scrutiny.
it's just to see if the kamaji cp gets templated and referenced correctly